modules/claims_api/config/schemas/v2/526.json

```diff
@@ -134,917 +135,1033 @@
-                },
-                {
-                    "if": {
-                        "properties": {
-                            "disabilityActionType": {
-                                "enum": ["NONE"]
-                            }
-                        },
-                        "required": ["secondaryDisabilities"]
-                    },
-                    "then": {
-                        "required": ["diagnosticCode"]
-                    }
+            "isRelatedToToxicExposure": {
+              "type": "boolean",
+              "description": "Is the disability related to toxic exposures? If so, include exposure information.",
+              "example": true
+            },
+            "exposureOrEventOrInjury": {
+              "type": "string",
+              "description": "Type of exposure, event, or injury.",
+              "examples": [
+                "Agent Orange",
+                "radiation",
+                "burn pits"
+              ]
+            }
+          },
+          "allOf": [
+            {
+              "if": {
+                "properties": {
+                  "disabilityActionType": {
+                    "enum": [
+                      "INCREASE"
+                    ]
+                  }
                 }
-            ]
+              },
+              "then": {
+                "required": [
+                  "ratedDisabilityId",
+                  "diagnosticCode"
+                ]
+              }
+            },
+            {
+              "if": {
+                "properties": {
+                  "disabilityActionType": {
+                    "enum": [
+                      "NONE"
+                    ]
+                  }
+                },
+                "required": [
+                  "secondaryDisabilities"
+                ]
+              },
+              "then": {
+                "required": [
+                  "diagnosticCode"
+                ]
+              }
+            }
+          ]
         }
-    },
-    "treatments": {
+      },
+      "treatments": {
         "description": "Identifies the Service Treatment information of the Veteran.",
         "type": "array",
         "minItems": 1,
         "uniqueItems": true,
         "items": {
-            "type": "object",
-            "additionalProperties": false,
-            "required": [
-                "treatedDisabilityNames",
-                "center"
-            ],
-            "properties": {
-                "beginDate": {
-                    "description": "Begin date for treatment. If treatment began from 2005 to present, you do not need to provide dates.",
-                    "type": ["null", "string"],
-                    "pattern": "^(0[1-9]|1[0-2])-(?:19|20)[0-9][0-9]$",
-                    "example": "02-2018"
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "treatedDisabilityNames",
+            "center"
+          ],
```

Comment:
Same question as above in relation to the required block here needing to be removed as per [API-16232](https://jira.devops.va.gov/browse/API-16232) ?